stories/Containers/ClientShowCase.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import styled from 'styled-components';
 import { Button } from '../../src/Inputs/Button';
+import { select } from '@storybook/addon-knobs';
```

Comment:
DO NOT USE ADDON KNOBS do not import this. Ths

Reply:
https://storybook.js.org/docs/react/api/argtypes#gatsby-focus-wrapper
read this